blaze-client/src/test/scala/org/http4s/client/blaze/Http1ClientStageSpec.scala

```diff
@@ -245,6 +245,28 @@ class Http1ClientStageSpec extends Specification {
       response must_==("done")
     }
 
+    "Not expect body if request was a HEAD request" in {
+      val contentLength = 12345L
+      val resp = s"HTTP/1.1 200 OK\r\nContent-Length: $contentLength\r\n\r\n"
+      val headRequest = FooRequest.copy(method = Method.HEAD)
+      val tail = mkConnection(FooRequestKey)
+      try {
+        val h = new SeqTestHead(List(mkBuffer(resp)))
+        LeafBuilder(tail).base(h)
+
+        val response = tail.runRequest(headRequest, false).run
+        response.contentLength must_== Some(contentLength)
+
+        // connection reusable immediately after headers read
+        tail.isRecyclable must_=== true
+
+        // body is empty due to it being HEAD request
+        response.body.runLog.run.foldLeft(0L)(_ + _.length) must_== 0L
```

Comment:
ah nice thanks, didn't know about that method but yes it looks good. I'd need an implicit `Monoid[Long]` in scope, do you know where I can import one of these or do I need to define it myself?
```
implicit val longMonoid = Monoid.instance[Long](_ + _, 0L)
```